src/tests/ftest/util/telemetry_utils.py

```diff
@@ -424,7 +424,7 @@ class TelemetryUtils():
         ENGINE_NVME_INTEL_VENDOR_METRICS
     ENGINE_MEM_USAGE_METRICS = [
         "engine_mem_vos_dtx_cmt_ent_48",
-        "engine_mem_vos_vos_obj_360",
+        "engine_mem_vos_vos_obj_384",
```

Comment:
What does 360 vs 384 represent?

Reply:
that's the size of 'vos_object' (a in-memory data structure), I don't know why the author put the size in the metric name though.